src/nevermined/utils/WebServiceConnector.ts

```diff
@@ -192,14 +206,48 @@ export class WebServiceConnector extends Instantiable {
         )
     }
 
+    public async fetchCID(cid: string): Promise<string> {
+        
+        const url = this.config.ipfsGateway + '/api/v0/cat?arg=' + cid.replace('cid://', '')
+        const authToken = WebServiceConnector.getIPFSAuthToken()
+        const options = {
+            method: 'POST',
+            ...(authToken && {
+            headers: { Authorization: `Basic ${authToken}` }
+            })
+        }
+
+        return fetch(url, options)
+            .then(async (res) => {
+                if (!res.ok) {
+                    throw new Error(
+                    `${res.status}: ${res.statusText} - ${await res.text()}`
+                    )
+                }
+                return res.text()
+            }).catch((err) => {
+                throw err
+            })
+    }
```

Comment:
should this just be added to the node? that way the sdk does not need filecoin or ipfs config options

Reply:
That would add a requirement of having a node running to resolve a DDO, and I don't think that has to be a tough requirement.